torchfix/common.py

```diff
@@ -73,6 +76,18 @@ def get_specific_arg(
                 return arg
         return None
 
+    @staticmethod
+    def has_specific_arg(
+            node: cst.Call, arg_name: str, position: Optional[int] = None
+    ) -> bool:
+        """
+        Check if a named argument is present in a call (not positional).
```

Comment:
> not positional
What is `position` passed?
I guess either delete `position` parameter or change the comment?